### PR TITLE
add vim.ui.select as a picker

### PR DESCRIPTION
**Pull request by Bleksak**
Hi, this is a PR to support vim.ui.select as a picker.
Later I will add support for fzf too

It works, but It's not switchable via config, you need to manually toggle it in default providers.
How would you want me to resolve this?

It would be great if we could add "picker" into configuration, however I don't see a way to do it now.

### CODE REVIEW

File: lua/laravel/ui_select/pickers/related.lua [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+          else
+            local relation = build_relation({ model_info[relation_type] }, relation_type)
+            if relation ~= nil then
+              table.insert(relations, relation)
+            end
+          end
+        end
+      end
+
+      print(vim.inspect(relations))
```
Comment (adalessa) [inline lua/laravel/ui_select/pickers/related.lua:79]: debug statement, I think needs to be removed.

File: lua/laravel/ui_select/pickers/resources.lua [hunk 2]
```diff
@@ -0,0 +1,46 @@
+local actions = require("laravel.ui_select.actions")
+-- local action_state = require("telescope.actions.state")
```
Comment (adalessa) [inline lua/laravel/ui_select/pickers/resources.lua:2]: remove line

**Comment by adalessa** [comment]
Thanks looks good, but I think there is duplication with the action between the pickers, I think adding this could be a good moment to extract them if they can be re-use.

**Comment by adalessa** [comment]
> It works, but It's not switchable via config, you need to manually toggle it in default providers.
How would you want me to resolve this?
It would be great if we could add "picker" into configuration, however I don't see a way to do it now.

Let me think about this and will get back to you on this.

**Comment by adalessa** [comment]
@Bleksak I have pushed a change to the branch that handles the configuration aspect of it. pls check it, if you have a question let me know.

**Comment by Bleksak** [comment]
@adalessa Thanks, I sucessfully integrated it and fixed the duplication issues. Can you check it now?

**Comment by adalessa** [comment]
Thanks for the collaboration, looks good

**Pull request merged**